### PR TITLE
Revert MPA circle CI build PRs for the release-7.5 branch

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Revert MPA circle CI build PRs for the release branch

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: app/platform/service_test.go [hunk 1]
```diff
@@ -124,7 +124,6 @@ func TestMetrics(t *testing.T) {
 	})
 
 	t.Run("ensure the metrics server is started with advanced metrics", func(t *testing.T) {
-		t.Skip("MM-47635")
```
Comment (agnivade) [inline app/platform/service_test.go:127]: This seems unrelated?

Comment (isacikgoz) [reply app/platform/service_test.go:127]: It was part of one of the PRs I cherry-picked the commit to fix the issue.

**Review by agnivade — APPROVED** [review]
